### PR TITLE
Evaluation

### PR DESCRIPTION
**Pull request by ardwennem2**
As a developer I want to write a evaluation of the functional modules and the product in its entirety, including the failure analysis. So that it can be added to the Final Report.

### CODE REVIEW

File: docs/final report/Evaluation.tex [hunk 1]
```diff
@@ -1,4 +1,21 @@
 \section{Evaluation}
+In this section an evaluation of the functionalities performed using a well-justified method will be presented, as well as a failure analysis – where the product does not perform as needed. This will be done for the functional modules, as well as for the product in its entirety. We will divide the functional modules into the building modules, and the modules for buying (land), sellin (land)g and demolishing (buildings).
```
Comment (HarmenKroon) [inline docs/final report/Evaluation.tex]: sellin this land, G

**Comment by jannelie** [comment]
Approved

**Comment by HarmenKroon** [comment]
The pdf is not correctly build

**Comment by HarmenKroon** [comment]
Approved

**Pull request merged**